README.md

```diff
@@ -0,0 +1,86 @@
+# Sandbox Engineering sb03
+
+The goal of this class is to prepare you to quickly build and scale products and systems.
+
+## 3 core parts of a product
+
+You can break down most software products into 3 parts:
+
+ - `Frontend` (the software on the user's device)
+ - `Backend` (the systems that you control)
+ - `Datastore` (anything that stores data)
+
+## The Stack
```

Comment:
Maybe a note here saying that this is the one we will be doing demos in, given I discuss a bunch of different stacks in lecture 3 (and how it's ultimately up to them what they can learn and work in fastest).